config/confighttp/confighttp.go

```diff
@@ -479,13 +480,14 @@ func (hss *ServerConfig) ToServer(_ context.Context, host component.Host, settin
 		next:            handler,
 		includeMetadata: hss.IncludeMetadata,
 	}
-
 	server := &http.Server{
 		Handler:           handler,
 		ReadTimeout:       hss.ReadTimeout,
 		ReadHeaderTimeout: hss.ReadHeaderTimeout,
 		WriteTimeout:      hss.WriteTimeout,
 		IdleTimeout:       hss.IdleTimeout,
+		// Setting the Server error logger
+		ErrorLog: zap.NewStdLog(settings.Logger),
```

Comment:
it needs to know to log at error level
```suggestion
		ErrorLog: zap.NewStdLog(settings.Logger, zapcore.ErrorLevel),
```